docs/src/main/asciidoc/_high-availability/subsystem-support/Kubernetes.adoc

```diff
@@ -0,0 +1,20 @@
+== Discovery for Kubernetes
+
+`KUBE_PING` is a discovery protocol for JGroups cluster nodes managed by Kubernetes.
+Since Kubernetes is in charge of launching nodes, it knows the IP addresses of all pods it started,
+and is therefore the best place to ask for cluster discovery.
+Discovery is therefore done by asking Kubernetes for a list of IP addresses of all cluster nodes.
+Combined with `bind_port` / `port_range`, the protocol will then send a discovery request to all instances and wait for the responses.
+
+To configure an existing profile with `KUBE_PING` use the following CLI batch replacing with appropriate stack:
+
+[source]
+----
+batch
+/subsystem=jgroups/stack=tcp/protocol=MPING:remove()
+/subsystem=jgroups/stack=tcp/protocol=kubernetes.KUBE_PING:add(add-index=0,properties={namespace="production",labels="cluster=nyc")}
```

Comment:
The `}` and `)` at the end of the line have been swapped here.

Reply:
Fixed, thanks. Also expanded the document a little while at it.